.eslintignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+build
+public
+cypress
```

Comment:
I think that we want to have eslint in cypress

Reply:
the that we can see cypress is here is not the "source code" of end-to-end tests but a folder with artifacts. But yea, it's confusing, I'll change the path of the test artifacts directory to not confuse

Reply:
Done